src/app/public/modules/popover/popover.component.ts

```diff
@@ -106,6 +106,8 @@ export class SkyPopoverComponent implements OnInit, OnDestroy {
 
   public classNames: string[] = [];
 
+  public dropdown = false;
```

Comment:
Is there a different name we could use for this boolean that makes sense in the context of popovers only?

Reply:
Another way to say it: is this property something that would be useful for a consumer of popovers?

Reply:
Absolutely. This was mostly a sanity check review. I've changed the property to `containOverflow`.